jbpm-container-test/pom.xml

```diff
@@ -46,6 +46,18 @@
     <version.org.jboss.remoting.naming>4.0.18.Final</version.org.jboss.remoting.naming>
   </properties>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
```

Comment:
Why are we skipping this? Currently it is not needed.

Reply:
@MarianMacik if I don't skip the enforcer plugin here - I got a bunch of transitive dependencies found and the build breaks

Reply:
@mbiarnes OK, why is it then passing currently on the master? Did we make any changes to enforcer in this PR?

Reply:
There were not activated all rules for all repositories.
I think for tests this is not big deal, but still it could be worth to clean and declare all deps you need and do not rely on transitive deps. So I would see this as temporary skipping and when it is merged we should look into why and if it still makes sense to skip it or not.

Reply:
OK, understood.